assets/local/local.js

```diff
@@ -1,9 +1,11 @@
-/**
- * Protect window.console method calls, e.g. console is not defined on IE
- * unless dev tools are open, and IE doesn't define console.debug
- * from http://stackoverflow.com/questions/3326650/console-is-undefined-error-for-internet-explorer#13817235
- */
 (function() {
+	"use strict";
```

Comment:
The real changes in this file are:

* Whole file wrapped in an instantly invoked function, rather than just the `window.console` polyfill.
* Remove `fixEditorWidth` method and call from `init`.
* Remove `userSort` method and call from `init`.
* The `resize`, `initDatepickers` and `initConditionRecommendationTypes` are unchanged and still called in `init`.
* Used short syntax for the call to `init` on load.